[WEB-3482] release (#425)
bitrise-io/bitrise-workflow-editor: README.md

### Run in development mode

1. __DOCKER__: In the Workflow Editor's directory, run `bitrise run up`. This uses `docker-compose` behind the scenes
1. __LOCAL__: You need to start the api `bitrise run up-api` and middleman  `bitrise run up-middleman-server`.

> 2. In your browser, you can reach the Workflow Editor on `localhost:1234`. Be aware that you usually have to wait a while until middleman starts up.

Reviewer: ```suggestion
1. In your browser, you can reach the Workflow Editor on `localhost:1234`. Be aware that you usually have to wait a while until middleman starts up.
```

> 3. By default, the Workflow Editor will open the bitrise.yml and .bitrise.secrets.yml found in this folder. For testing purposes, you probably want to be able to edit custom files. This can be achieved by setting the `TEST_BITRISE_CONFIG_PATH` and `TEST_BITRISE_SECRETS_PATH` environment variables with the path pointing to the custom files' paths.

Reviewer: ```suggestion
1. By default, the Workflow Editor will open the bitrise.yml and .bitrise.secrets.yml found in this folder. For testing purposes, you probably want to be able to edit custom files. This can be achieved by setting the `TEST_BITRISE_CONFIG_PATH` and `TEST_BITRISE_SECRETS_PATH` environment variables with the path pointing to the custom files' paths.
```

### Run client tests

Use `npm test` for a single run or `npm run test-watch` for a continous test execution. __Note__ the latter option might have some delays since middleman needs to compile assets upon every change before karma runner kicks in.

If you only iterate on tests, you can also use `npm run karma` as it skips middleman and the transpilation and run the tests on an already transpiled JS. (faster)

# Contributing

This project is using squash & merge model, feel free to have as many commits as you like but at the end the work will end up on master as a single commit.

> **IMPORTANT** Plese make sure every commit on the master is released, there are no expections. Someone from bitrise should be able to assist with that!

Reviewer: ```suggestion
**IMPORTANT** Please make sure every commit on the master is released, there are no exceptions. Someone from Bitrise should be able to assist with that!
```

> If you are planning to break down the tasks into multiple master commit (aka milestones) please use a long-living feature branch and fork that.

Reviewer: ```suggestion
If you are planning to break down the tasks into multiple master commits (aka milestones) please use a long-living feature branch and fork that.
```
